packages/gatsby-source-wordpress/src/steps/source-nodes/fetch-nodes/fetch-nodes-paginated.js

```diff
@@ -84,9 +84,11 @@ const paginatedWpNodeFetch = async ({
   }
 
   let {
-    [contentTypePlural]: { nodes, pageInfo: { hasNextPage, endCursor } = {} },
+    [contentTypePlural]: { nodes, pageInfo },
```

Comment:
these look almost the same. Where this is different is we never get to the `= {}` value fallback since `null` is a value.